calculator.go

```diff
@@ -1,13 +1,135 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes two or more numbers and returns the result of adding them together.
+func Add(args ...float64) float64 {
+	var result float64
+
+	for _, arg := range args {
+		result += arg
+	}
+
+	return result
 }
 
-// Subtract takes two numbers and returns the result of subtracting the second
+// Subtract takes two or more numbers and returns the result of subtracting the second
 // from the first.
-func Subtract(a, b float64) float64 {
-	return b - a
+func Subtract(args ...float64) float64 {
+	var result float64
+
+	for i, arg := range args {
+		if i == 0 {
+			result = arg
+			continue
+		}
+		result -= arg
+	}
+
+	return result
+}
+
+//Multiply takes two or more numbers and returns the result of multiplying the second
```

Comment:
It's a good idea to always leave a space after the comment delimiter, so that you can never write a compiler pragma by accident! (For example, `go:embed...`).